api/src/com/cloud/network/Network.java

```diff
@@ -243,7 +245,7 @@ public static Capability getCapability(String capabilityName) {
 
         Allocated("Indicates the network configuration is in allocated but not setup"), Setup("Indicates the network configuration is setup"), Implementing(
                 "Indicates the network configuration is being implemented"), Implemented("Indicates the network configuration is in use"), Shutdown(
-                "Indicates the network configuration is being destroyed"), Destroy("Indicates that the network is destroyed");
+                        "Indicates the network configuration is being destroyed"), Destroy("Indicates that the network is destroyed");
```

Comment:
This appears to be an unnecessary formatting change that actually misaligns the code.